user-manual/administer/manage-jobs.rst

```diff
@@ -162,12 +162,14 @@ for startDate, endDate, jobName, jobStatus, jobInfo, and jobUser.
 **Clear inactive jobs**
 
 Clicking this button will remove any inactive jobs regardless of status (i.e.
-completed or failed jobs) created by you from the Jobs page. Any jobs still
+completed or failed jobs) created by you from the Jobs page. This includes
+inactive jobs that you inititiated via command-line in a terminal. Any jobs still
 processing, or jobs created by another user, will not be affected.
 
 .. NOTE::
 
-   You can **only** clear jobs that you have initiated.
+   You can **only** clear jobs that you have initiated via the web-based
+   dashboard and via the command-line in a terminal.
```

Comment:
I think that only an admin can clear CLI-related jobs?

Reply:
Yes, only admins can clear CLI-related jobs. Will update, thx!